pages/coinbase/homepage.py

```diff
@@ -4,7 +4,6 @@
 from selenium import webdriver
 from selenium.webdriver.common.keys import Keys
 
```

Comment:
I didn't scrub everything in homepage.py,  but I did take out all of the unnecessary attributes. Let me know if you want me to scrub these small edits.